reference/src/util/include/common.h

```diff
@@ -40,11 +39,20 @@
 #define EC_ED448_KEY_SIZE 57
 #define EC_X448_KEY_SIZE 56
 #define GCM_IV_LENGTH 12
+#define DH_768_BYTE_LENGTH 96L
+#define DH_1024_BYTE_LENGTH 128UL
+#define DH_1536_BYTE_LENGTH 192UL
+#define DH_2048_BYTE_LENGTH 256UL
+#define DH_3072_BYTE_LENGTH 384UL
+#define DH_4096_BYTE_LENGTH 512UL
 #define DH_MAX_MOD_SIZE 512
 #define RSA_PKCS1_PADDING_SIZE 11
 #define RSA_OAEP_PADDING_SIZE 42
 #define CHACHA20_NONCE_LENGTH 12
 #define CHACHA20_COUNTER_LENGTH 4
 #define CHACHA20_TAG_LENGTH 16
+#define MAX_NUM_SLOTS 256
+
+#define DEFAULT_ROOT_KEYSTORE_PASSWORD "@UJ2@&69&3vsW2@8lg#r4#AU2&Zzv5"
```

Comment:
Recommend adding comment around the intent of this password and why there are no security concerns with embedding it in source files.

Reply:
Added to readme.